core-backend/sandbox/src/funcs.rs

```diff
@@ -338,6 +338,14 @@ where
         })
     }
 
+    pub fn read_unchecked(ctx: &mut Runtime<E>, args: &[Value]) -> SyscallOutput {
```

Comment:
we don't need separate syscall for this
use already existing one by playing around value of "resulting" ptr arg: if u32::MAX than mean as infallible